docs/getting_started.md

```diff
@@ -118,10 +118,10 @@ kubectl describe pod busybox-curl-7bd5767999-spf7v
     Image: ghcr.io/open-feature/flagd:v0.4.4
     Port: 8014/TCP
     Host Port: 0/TCP
-    Args:
-      start
-      --uri
-      core.openfeature.dev/default/featureflagconfiguration-sample
+    -args:
+      - start
+      - --sources
+      - '[{"uri":"core.openfeature.dev/default/featureflagconfiguration-sample","provider":"kubernetes"}]'
```

Comment:
```suggestion
      - '[{"uri":"default/featureflagconfiguration-sample","provider":"kubernetes"}]'
```

Reply:
We don't need to use the prefix here as we are already defining the provider type as Kubernetes

Reply:
Yes, we can support both usages [1]. So we are getting rid of the prefix, we need a flagd release first. 

[1] - https://github.com/open-feature/flagd/blob/dc9cb5fa2005630fd68c7bc0a63a2b056430ff0d/core/pkg/runtime/from_config.go#L192